jib-core/src/main/java/com/google/cloud/tools/jib/api/JavaContainerBuilder.java

```diff
@@ -74,6 +53,20 @@ public static JavaContainerBuilder fromDistroless() throws InvalidImageReference
     return from(RegistryImage.named("gcr.io/distroless/java"));
   }
 
+  /**
+   * Creates a new {@link JavaContainerBuilder} that uses distroless jetty as the base image. For
+   * more information on {@code gcr.io/distroless/java}, see <a
+   * href="https://github.com/GoogleContainerTools/distroless">the distroless repository</a>.
+   *
+   * @return a new {@link JavaContainerBuilder}
+   * @throws InvalidImageReferenceException if creating the base image reference fails
```

Comment:
Maybe we can shallow this exception. It will never be thrown, because `gcr.io/distroless/java/jetty` is always a valid reference.